INTITA/Models/OpportunitiesType.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  OpportunitiesModel.swift
```

Comment:
You can rename the comment too, but it's minor. And replace this file from Models into Modules/OpportunitiesScreen

Reply:
Sorry. My bad. I renamed it.